src/Bridge/Doctrine/Common/Filter/AbstractFilter.php

```diff
@@ -11,16 +11,13 @@
 
```

Comment:
As this class isn't marked as internal, we should deprecate the old one. The old class can extend the new one.